### PR TITLE
Implement ynh_docker_image_extract

### PR DESCRIPTION
**Pull request by yalh76**
based on https://github.com/jjlin/docker-image-extract/pull/1

### CODE REVIEW

**Comment by yalh76** [comment]
Todo
Add a vendor with the script
ynh_secure_remove instead of rm
using app.src extending ynh_setup_source

File: ynh_docker_image_extract/ynh_docker_image_extract [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	fi
+
+	tempdir="$(mktemp -d)"
+	pushd $tempdir
+		git init -q
+		git remote add -f -t main origin https://github.com/jjlin/docker-image-extract.git > /dev/null 2>&1
+		git checkout -q -b main origin/main
+		./docker-image-extract -p $os_arch_variant -o $dest_dir $image_spec 2>&1
+	popd
+	rm -rf $tempdir
```
Comment (zamentur) [inline ynh_docker_image_extract/ynh_docker_image_extract]: use ynh_secure_remove

Comment (yalh76) [reply ynh_docker_image_extract/ynh_docker_image_extract]: done

**Comment by yalh76** [comment]
no more needed, as implemented in the core

**Pull request closed without merging**